cmd/dex/serve.go

```diff
@@ -6,6 +6,7 @@ import (
 	"crypto/x509"
 	"errors"
 	"fmt"
+	"google.golang.org/grpc/reflection"
```

Comment:
Please put that into a section with the other 3rd-party imports :)

Reply:
Sorry, I IDE did that automatically for me and I wasn't paying attention - sure